common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -338,7 +338,12 @@ def manage_video_subtitles_save(item, user, old_metadata=None, generate_translat
 
     # 1.
     html5_ids = get_html5_ids(item.html5_sources)
-    possible_video_id_list = [item.youtube_id_1_0] + html5_ids
+
+    # According to check_transcripts in contentstore/views/transcripts_ajax.py, transcript for youtube
```

Comment:
I think making this simpler would be better than technical details, may be something like this:
``` 
Youtube transcript source should always have a higher priority than html5 sources. Appending `youtube_id_1_0` at the end helps achieve this when we read transcripts list. 
```